### ISSUE TITLE
Hypercompliant tympanic membranes

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11509]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
Tympanometry reflects change in physical properties of the middle ear system as air pressure in the external ear canal is systematically varied. 

PMID: 32901364  Tympanometry results were assessed on an individual ear basis. Results were available for 90 ears. Forty-two (47%) tympanic membranes demonstrated normal type A tympanograms, 10 (11%) were classified as type B, 12 (13%) had negative middle ear pressure (type C) and 4 (5%) had perforations; 22 (24%) of examined **tympanic membranes were hypercompliant** (type Ad).
(...) 24% of patients were found to have hypermobile tympanic membranes; this is lower than the incidence of 36–46% reported in type 1 Stickler Syndrome patients [15, 29]. Otology questionnaire responses suggested that tympanic membrane hypermobility was not acquired through infection-related trauma. It is hypothesised that mutated COL11A1 fibrils are likely to be present in the fibrous layer of the tympanic membrane, where other collagen types have been demonstrated to be present. This motivates further investigation of primary collagen defects present in the tympanic membranes of patients with type 2 Stickler Syndrome and further investigation into the correlation between joint hypermobility and hypercompliant tympanic membranes.

PMID: 28306229 in the The 2017 International Classification of the Ehlers–Danlos Syndromes, **Hypercompliant tympanic membranes** is a minor criteria of Brittle Cornea Syndrome

PMID: 23642083 Where tympanometry has been performed, hyper-compliant tympanic membranes, as may be seen also in association with other generalised connective tissue disorders, have been demonstrated in a very high proportion of patients. Hypercompliant tympanic membranes are thought to be caused by a combination of factors, including increased intrinsic compliance of the eardrum itself, and joint laxity between the ossicles of the middle ear. Tympanometry is a non-invasive test routinely available in audiological medicine and may therefore be a useful diagnostic procedure to assess the likelihood of BCS. The identification of hypercompliant tympanic membranes should prompt further consideration of BCS as a possible diagnosis. No evidence of hearing loss or abnormal tympanometry in heterozygous individuals has yet been identified.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Brittle Cornea Syndrome  ORPHA:90354 
Stickler syndrome type 2 ORPHA:90654

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
This is a synonym of [Tympanic membrane hypermobility HP:6000015](https://hpo.jax.org/browse/term/HP:6000015). The hypercompliance means they move more. I will add as a synonym.

Hypercompliant tympanic membranes

**Comment by pnrobinson** [comment]
done
